core/server/src/main/webapp/browse-body.jsp

```diff
@@ -38,6 +65,10 @@
               <% } %>
             </ul>
```

Comment:
Thanks for the update! I suggest we add the input inside `<div class="navbar-inner">`. This will help consolidate the information and place it where it more makes sense.

This html will do the trick:

```
<div id="pathNav" class="input-append pull-right" style="margin: 5px;margin-right: 45px;">
    <input class="span12" id="pathInput" type="text" placeholder="Navigate to a directory">
    <button class="btn" type="button">Go</button>
</div>
```

![screen shot 2017-01-06 at 2 16 31 pm](https://cloud.githubusercontent.com/assets/1802050/21734952/1893eaaa-d41b-11e6-84c2-bf79aa2db855.png)